runCrawler.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/sh
 if [ $# -eq 2 ]; then
-    mvn compile -Pcrawler -DproxyHost="$1" -DproxyPort="$2"
+    mvn validate -Pcrawler -DproxyHost="$1" -DproxyPort="$2"
+elif [ $# -ne 2 ] || [ $# -ne 0 ]; then
```

Comment:
```suggestion
elif [ $# -ne 2 ] && [ $# -ne 0 ]; then
```

Though honestly, it'd be better to just make this the case for 0 arguments (`elif [ $# -eq 0 ]; then`), and put the error in the final `else` clause.

Reply:
I have made the required changes